README.md

```diff
@@ -650,6 +649,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Open Government, New Zealand](https://www.data.govt.nz/) | New Zealand Government Open Data | No | Yes | Unknown |
 | [Open Government, Peru](https://www.datosabiertos.gob.pe/) | Peru Government Open Data | No | Yes | Unknown |
 | [Open Government, Romania](http://data.gov.ro/) | Romania Government Open Data | No | No | Unknown |
+| [Open Government, Russia](https://data.gov.ru/?language=en) | Open Data Portal Russia | Unknown | No | Unknown |
```

Comment:
- auth value is not enclosed with `backticks`
- Unknown is not a valid Auth option. 

Currently, the only accepted inputs for the Auth field are as follows:

`OAuth` - the API supports OAuth
`apiKey` - the API uses a private key string/token for authentication - try and use the correct parameter
`X-Mashape-Key` - the name of the header which may need to be sent
`User-Agent` - the name of the header to be sent with requests to the API
No - the API requires no authentication to run

This API use apiKey for auth: https://data.gov.ru/pravila-i-rekomendacii

```suggestion
| [Open Government, Russia](https://data.gov.ru/?language=en) | Open Data Portal Russia | `apiKey` | No | Unknown |
```

Reply:
Thanks for this comment. 👍